libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -71,6 +71,22 @@ import {
   BooleanInput,
 } from '@angular/cdk/coercion';
 
+/**
+ * @internal Output type for hoverStart abnd hoverEnd attributes.
+ * Contains the main data of the hovered series, and wether it was hovered in the stack or on the legend.
+ * When hovered over the stack, it shows the data for the hovered stack as well as the hovered series.
+ * When hovered over the legend, it only shows the data for the hovered series.
+ *  */
+export type DtStackedSeriesChartHoverData = {
```

Comment:
Can we please provide more type info here as mentioned on slack. 

The properties stackName, value and selected are only undefined when the legend is hovered instead of the stack.

Reply:
Just saw your comment on Slack, will apply your suggestion right away